service/src/main/java/org/cbioportal/service/impl/MolecularDataServiceImpl.java

```diff
@@ -216,6 +216,7 @@ private void validateMolecularProfile(String molecularProfileId) throws Molecula
         MolecularProfile molecularProfile = molecularProfileService.getMolecularProfile(molecularProfileId);
 
         if (molecularProfile.getMolecularAlterationType().equals(MolecularAlterationType.MUTATION_EXTENDED) || 
+            molecularProfile.getMolecularAlterationType().equals(MolecularAlterationType.MUTATION_UNCALLED) ||
```

Comment:
MUTATION_UNCALLED shouldn't be used for `molecular-data`, only used for `mutations`